.github/pull_request_template.md

```diff
@@ -1,5 +1,8 @@
 ### Pull Request checklist ###
 <!-- Before submitting the PR, please address each item -->
+- **Breaking changes**:  This PR follows our [breaking change policy](https://github.com/mozilla/application-services/blob/main/docs/howtos/breaking-changes.md)
+  - [ ] This PR has no breaking API changes
+  - [ ] There are corresponding PRs for our consumer applications that resolve the breaking changes and have been approved
```

Comment:
very nano nit: maybe having only as one high level `[ ]` and two nested dot items since it's always going to be either or - something like:
```suggestion
  - [ ] This PR follows the breaking change policy:
     - This PR has no breaking API changes, or
     - There are corresponding PRs for our consumer applications that resolve the breaking changes and have been approved
```